charts/vh-booking-queue-subscriber/values.yaml

```diff
@@ -0,0 +1,39 @@
+---
+java:
+  image: 'sdshmctspublic.azurecr.io/vh/booking-queue-subscriber:20210603.9'
+  applicationPort: 8080
+  replicas: 2
+  ingressHost: vh-booking-queue-subscriber.{{ .Values.global.environment }}.platform.hmcts.net
+  releaseNameOverride: vh-booking-queue-subscriber
+  readinessPath: /health/liveness
+  livenessPath: /health/liveness
+  livenessDelay: 360
+  livenessPeriod: 120
+  aadIdentityName: vh-aad-identity
+  keyVaults:
+    vh-infra-core:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - applicationinsights--instrumentationkey
+        - ServiceBusConnection
+    vh-booking-queue:
+      excludeEnvironmentSuffix: false
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - azuread--clientsecret
+        - azuread--clientid
+  environment:
+    ASPNETCORE_URLS: http://+:8080
+    AZUREAD__TENANTID: "fb6e0e22-0da3-4c35-972a-9d61eb256508"
```

Comment:
Tenant Id hardcoded

Reply:
yep, I don't think that's an issue though? Tenant ID's are not secrets

Reply:
Oh, and that's the tenant for AAD